scripts/extensions/auto-tagging-widget/src/auto-tagging.tsx

```diff
@@ -388,10 +399,20 @@ export function getAutoTaggingComponent(superdesk: ISuperdesk, label: string) {
 
                                 const others = items.filter((tag) => isEntity(tag) === false);
                                 const othersGrouped = others.groupBy((tag) => tag.group.value);
-                                const othersGroupedAndSorted = othersGrouped.sortBy(
-                                    (_, key) => this.iMatricsFields[key].order ?? -1,
+
+                                const othersGroupedAndSortedByConfig = othersGrouped
+                                    .filter((_, key) => !!this.iMatricsFields[key])
+                                    .sortBy((_, key) => this.iMatricsFields[key].order,
```

Comment:
You are only passing `comparatorValueMapper` here. It'd be good to pass comparator function too, so it's clear how items are compared. The function should be `(a, b) => a - b`